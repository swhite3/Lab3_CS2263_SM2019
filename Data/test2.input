This is a very long test
