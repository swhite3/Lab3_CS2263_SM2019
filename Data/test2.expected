test long very a is This
